pkg/consts/consts.go

```diff
@@ -87,8 +87,9 @@ const (
 	// Used by the home chain reader.
 	MethodNameGetCapability = "GetCapability"
 
-	// CCIPConfig.sol methods.
+	// CCIPHome.sol methods.
 	// Used by the home chain reader.
+	// TODO: change them to getConfig, getAllConfigs
```

Comment:
This should not be a blocker for now, but lets create a followup ticket for it.

Reply:
https://smartcontract-it.atlassian.net/browse/CCIP-3642